maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/dto/GenericDTO.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.laa.maat.orchestration.dto;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+public abstract class GenericDTO implements Serializable {
+    private Timestamp timestamp;
+    private Boolean selected;
+    private Boolean mDirty;
+
+    public GenericDTO() {
+    }
+
+    public Timestamp getTimestamp() {
+        return this.timestamp;
+    }
+
+    public void setTimestamp(Timestamp timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public boolean isSelected() {
+        return this.selected != null ? this.selected : false;
+    }
+
+    public void setSelected(boolean selected) {
+        this.selected = Boolean.valueOf(selected);
+    }
+
+    public boolean isDirty() {
+        return this.mDirty != null ? this.mDirty : false;
+    }
+
+    public void setDirty(boolean dirty) {
+        this.mDirty = Boolean.valueOf(dirty);
+    }
+
+    public abstract Object getKey();

```

Comment:
We should avoid using raw types, can you replace `Object` with a generic type `T`?

https://docs.oracle.com/javase/tutorial/java/generics/types.html
e.g.
```
/**
 * Generic version of the Box class.
 * @param <T> the type of the value being boxed
 */
public class Box<T> {
    // T stands for "Type"
    private T t;

    public void set(T t) { this.t = t; }
    public T get() { return t; }
}
```